code/modules/reagents/chemistry/reagents/medicine.dm

```diff
@@ -608,6 +610,44 @@
 		M.visible_message("<span class='notice'>[M] looks a bit dazed.</span>")
 	return ..()
 
+/datum/reagent/medicine/diphenhydramine/overdose_process(mob/living/M, severity)
+	var/list/overdose_info = ..()
+	var/effect = overdose_info[REAGENT_OVERDOSE_EFFECT]
+	var/update_flags = overdose_info[REAGENT_OVERDOSE_FLAGS]
+	if(severity == 1)
+		if(effect <= 2)
+			to_chat(M, "<span class='warning'>You feel parched.</span>")
+		else if(effect <= 3)
+			to_chat(M, "<span class='warning'>You feel a little off.</span>")
+			M.Dizzy(10 SECONDS)
+		else if(effect <= 5)
+			to_chat(M, "<span class='warning'>You feel a sudden head rush.</span>")
+			M.emote("faint")
+		else if(effect <= 8)
+			M.Druggy(30 SECONDS)
+	else if(severity == 2)
```

Comment:
```suggestion
			M.Druggy(30 SECONDS)

	else if(severity == 2)
```
I prefer my big `if else if` chains to be broken up like this, for extra readability. Feel free to ignore this however